toonz/sources/toonzlib/stage.cpp

```diff
@@ -69,8 +69,8 @@ typedef std::vector<Player> PlayerSet;
    thickness of
                 images .pli.
 */
-const double Stage::inch      = 53.33333;  // consider changing to 120 or 160
-const double Stage::vectorDpi = 53.3333;
+const double Stage::inch        = 53.33333;  // consider changing to 120 or 160
```

Comment:
I would take out the "consider changing comments" - this would confuse future developers